packages/generator-react-server/test/app.js

```diff
@@ -129,7 +130,7 @@ function installDeps() {
 			} else {
 				const localDeps = ['react-server-cli', 'react-server']
 					.map(dep => path.resolve(path.join(__dirname, '../..', dep)));
-				cp.exec('npm install ' + localDeps.join(' '), (error) => {
+				cp.exec(shellescape(['npm', 'install'].concat(localDeps)), (error) => {
```

Comment:
Out of curiosity, is there a deps that needs escaping, or is this defensive in case we add one? I think the change is good either way